safety_gym/search/planner/maze.py

```diff
@@ -0,0 +1,48 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+
+from shrl.config.planner_config import planning_config
+
+
+class Maze:
+    def __init__(self, task_name: str):
+        self.maze_config = planning_config[task_name]
+        self.maze_np = np.zeros(self.maze_config["board_size"])
+        self._build()
+
+    def plot(self):
```

Comment:
Could you try this function on your results and double-check the maze you build is right. Hope to get a snapshot of the running result on this function.